plugins/nodes/src/js/components/NodesGridDials.js

```diff
@@ -43,9 +43,11 @@ var NodesGridDials = React.createClass({
     }
 
     Object.keys(resourcesByFramework).forEach(function(frameworkID) {
-      var percentage =
-        resourcesByFramework[frameworkID][props.selectedResource] * 100;
-      percentage /= node.getUsageStats(props.selectedResource).total;
+      const used =
+        resourcesByFramework[frameworkID][props.selectedResource] || 0;
+      const total = node.getUsageStats(props.selectedResource).total || 0;
+
+      const percentage = total === 0 ? 0 : used * 100 / total;
```

Comment:
Doesn't this need a Math.round?

Reply:
We're rounding the data in https://github.com/dcos/dcos-ui/pull/2642/files/30b65a9a45a0e6029c6e6d9d779d7d26cefb8192#diff-aee5c9bb5a89d3f105fa091ea847c99bR101